internal/interface/memory/linkrepo/memory.go

```diff
@@ -1,6 +1,7 @@
-package linkstorage
+package linkrepo
 
 import (
+	link2 "koro.che/internal/domain/link"
```

Comment:
You do like math, don't you? Using numbering in variable names doesn't convey any sensible meaning, though..